rpcapd/CMakeLists.txt

```diff
@@ -136,7 +136,12 @@ if(WIN32 OR ((CMAKE_USE_PTHREADS_INIT OR PTHREADS_FOUND) AND HAVE_CRYPT))
   set(MANFILE_EXPAND rpcapd-config.manfile.in)
 
   if(NOT MSVC)
-    install(TARGETS rpcapd DESTINATION sbin)
+    # DLL and EXE need to be in same bin path in Win32 platform
+    if(MINGW)
+      install(TARGETS rpcapd DESTINATION ${CMAKE_INSTALL_BINDIR})
+    else(MINGW)
+      install(TARGETS rpcapd DESTINATION sbin)
+    endif(MINGW)
```

Comment:
It would be nice if this old and ugly syntax would be dropped in favour of:
```cmake
if(MINGW)      
  install(TARGETS rpcapd DESTINATION ${CMAKE_INSTALL_BINDIR})    
else()
  install(TARGETS rpcapd DESTINATION sbin)    
endif()
```

But what is `sbin`? I have MinGW (TDM-gcc), but no `sbin` among it's sub-dirs.

Reply:
I choose that way not to create issue in other environment.

Reply:
> But what is `sbin`? I have MinGW (TDM-gcc), but no sbin among it's sub-dirs.

It's a UN\*X directory...

...but it's not necessarily the right one for daemons.

I no longer remember the exact rationale for the name; it dates back to a proposal by some people at Sun in the mid-1980's.  That proposal was originally done for SunOS 4.0, with at least part of the rationale being to better support diskless workstations without using Sun's ND protocol for accessing a file as a network disk.  [The Wikipedia page on the (mostly Linux) Filesystem Hierarchy Standard](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) (FHS), a spec influenced by predecessors dating back to the aforementioned proposal, says /sbin and /usr/sbin are for "system binaries", so I guess that's what "sbin" stands for.  (See [somebody's 1992 USENET posting](https://groups.google.com/g/comp.unix.questions/c/g9DsvKQx8h8/m/QNs0F-mHpR4J) for a bit more history.)

The FHS says /usr/sbin is for daemons, but some other systems use /usr/libexec instead.

In any case, with this change, the CMakeLists.txt file uses it for *non*-MSVC, *non*-MinGW environments.

Reply:
You may want to use `${CMAKE_INSTALL_SBINDIR}` instead of `${CMAKE_INSTALL_BINDIR}` and `sbin`

See https://cmake.org/cmake/help/latest/command/install.html#installing-files

Reply:
As I said in the commit message, DLL and exe need to be in same folder I.e. bin.

Reply:
I've changed that part of rpcapd/CMakeLists.txt to make the sbin-or-not decision based on whether the target is Windows rather than on whether the compiler is MSVC.  You'll need to rebase and update the change.